standard/links.lua

```diff
@@ -31,7 +31,7 @@ local _PREFIXES = {
 	esl = '',
 	facebook = 'http://facebook.com/',
 	['facebook-gaming'] = 'https://fb.gg/',
-	faceit = 'https://www.faceit.com/en/players/',
+	faceit = '',
```

Comment:
do we have any usage of faceit besides for players and teams?

Reply:
I believe we also have cites and brackets which is under `https://www.faceit.com/en/championship/` any brackets are based off this core url, then the events specific url, then `/standings/` so we dont need to worry about having different entries for the main site and the brackets

Reply:
cites and brackets do not use this module
if there are not any other usages besides infobox player, team and league we could set the default with the link for league (since player and team have their variants)

Reply:
example from CS infobox league:
```{{#if:{{{faceit|}}}|[https://www.faceit.com/en/csgo/tournament/{{{faceit}}} <i class="lp-icon lp-faceit"></i>]}}<!--
--> {{#if:{{{faceit-c|}}}|[https://www.faceit.com/en/championship/{{{faceit-c}}}/event <i class="lp-icon lp-faceit"></i>]}}<!--
--> {{#if:{{{faceit-c2|}}}|[https://www.faceit.com/en/championship/{{{faceit-c2}}}/event <i class="lp-icon lp-faceit"></i>]}}<!--
--> {{#if:{{{faceit-hub|}}}|[https://www.faceit.com/en/hub/{{{faceit-hub}}}/ <i class="lp-icon lp-faceit"></i>]}}<!--
--> {{#if:{{{faceit-org|}}}|[https://www.faceit.com/en/organizers/{{{faceit-org|}}}/ <i class="lp-icon lp-faceit"></i>]}}```

Reply:
example from R6 infobox league:
```{{#if:{{{faceit|}}}|[{{{faceit}}} <i class="lp-icon lp-faceit"></i>]}}```

Reply:
we should decide on one standard, adjust all wikis to it (via bots)

Reply:
see #877